lib/database.js

```diff
@@ -54,6 +54,9 @@ api.localCollection = null;
 // local document ID
 api.localDocumentId = 'local';
 
+// migration API
+api.migration = require('./migration');
```

Comment:
If we think this is an unstable API right now... perhaps we should expose it as `api._migration` until we're more comfortable with it. Thoughts? @mattcollier @davidlehn